src/generic-site/content-script.js

```diff
@@ -34,5 +35,38 @@ function updateSettings(new_settings = null) {
     }
 }
 chrome.runtime.onMessage.addListener(handleMessage);
+
+function sendRollToGameLog(message) {
+    console.log(message)
+    chrome.runtime.sendMessage(message)
+}
+
+function disconnectAllEvents() {
+    registered_events.forEach((event) => {
+        event.disconnect();
+    })
+}
+
+function handleIncomingSettings({ character, action, type} = {}) {
+    characterInternal = character;
+    chrome.runtime.sendMessage({
+        action: "hp-update",
+        character
+    })
+    chrome.runtime.sendMessage({
+        action: "settings",
+        type: "character",
+        settings: character.settings,
+        id: character.id
+    })
+}
+
+var registered_events = [];
+registered_events.push(addCustomEventListener("rendered-roll", sendRollToGameLog));
```

Comment:
Technically, this sends a message whether it's a `roll` or `rendered-roll` or `hp-update` makes no difference. The function name shouldn't be "sendRollToGameLog" either since it's sending to roll20, not the game log.
So I think the event should be `addCustomEventListener("send-message", sendMessageToBeyond20)` instead.